smmips/smmips.py

```diff
@@ -169,78 +169,71 @@ def assign_smmips(outdir, sortedbam, prefix, remove, panel, upstream_nucleotides
         json.dump(smmip_counts, newfile, indent=4)
    
 
-def merge_chromosome_files(outdir, remove):
+def merge_chromosome_files(prefix, files, file_type, remove):
```

Comment:
maybe the name of the function should be change to just merge_files?    we may generate multiple files per chromosome and in theory could have files that could be generated from regions that span multiple chromosomes.

Reply:
agreed. this is a left-over of the previous implementation when the panel was broken up into full length chromosomes.